docs/tutorials/Healthcare - Digitizing the communication/3-onboarding-doctor.md

```diff
@@ -1,19 +1,24 @@
 ---
-sidebar_position: 2
+sidebar_position: 4
 ---
 
-# Welcome user
+# Welcome doctor
 
 So, let's go through the code we prepared for you. We start from onboarding the users in our system. 
+
```

Comment:
We need to separate the part where we explain the code we provided in the initial branch and the other part where the readers ought to implement parts of the code.